applications/GeoMechanicsApplication/custom_strategies/schemes/geomechanics_time_integration_scheme.hpp

```diff
@@ -25,6 +25,19 @@ class GeoMechanicsTimeIntegrationScheme : public Scheme<TSparseSpace, TDenseSpac
     using LocalSystemVectorType = typename BaseType::LocalSystemVectorType;
     using LocalSystemMatrixType = typename BaseType::LocalSystemMatrixType;
 
+    int Check(const ModelPart& rModelPart) const override
+    {
+        KRATOS_TRY
+
+        Scheme<TSparseSpace, TDenseSpace>::Check(rModelPart);
```

Comment:
We can make this slightly more readable:
```suggestion
        BaseType::Check(rModelPart);
```

Reply:
I actually prefer to keep it like this: now it is clear at a glance (without having to look at the definition) which function is called. We could discuss (we could also delete the base check, since at this moment it just returns 0 and funnily enough has a KRATOS_TRY and CATCH around it).